setup.py

```diff
@@ -79,12 +79,13 @@
     author_email="snapcraft@lists.snapcraft.io",
     url="https://github.com/canonical/craft-providers",
     classifiers=[
-        "Development Status :: 2 - Pre-Alpha",
+        "Development Status :: 4 - Beta",
```

Comment:
is beta to be removed once adopted by snapcraft?

Reply:
My plan was to promote it to stable once snapcraft reaches production. We may need to fix issues revealed during the snapcraft tests.